portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +31,69 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  
+  /**
+   * Takes in a query object and places its content in a json formatted string
+   * and returns it.
+   */
+  private String getJson(Query commentQuery){
+    String returnStr = "{";
+    int count = 0;
+
+    DatastoreService dataStore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery prepCommentQuery = dataStore.prepare(commentQuery);
+    
+    for(Entity commentEntity : prepCommentQuery.asIterable()){
+      returnStr += "\"comment"+ Integer.toString(count)+"\": " + "\"" + 
+        commentEntity.getProperty("content") + "\", ";
+      count++;
+    }
+
+    //Lobbing off extra characters from the end of string if necessary
```

Comment:
Nit: space between // and first word of comments. Another nit is to use '.' (Google style guide)

Reply:
Here and below.